catalyst/data/dataset.py

```diff
@@ -1,9 +1,9 @@
-from typing import Any, Callable, Dict, List, Union  # isort:skip
 from pathlib import Path
+from typing import Any, Callable, Dict, List, Union
```

Comment:
Could you please return `# isort:skip` and move `from typing import ...` line to the top of the file?

Reply:
done